get-cabal-configuration.hs

```diff
@@ -447,6 +447,10 @@ getSourceDirectories :: [BuildInfo] -> FilePath -> [String]
 getSourceDirectories buildInfo cabalDir =
     map (cabalDir </>) (concatMap hsSourceDirs' buildInfo)
 
+getIncludeDirectories :: [BuildInfo] -> FilePath -> [String]
+getIncludeDirectories buildInfo cabalDir =
+  map (cabalDir </>) (concatMap includeDirs buildInfo)
```

Comment:
Please use 4 spaces indentation to keep style consistent with the rest of the file.

Reply:
Ah sorry, fixed now.